alphadia/outputtransform.py

```diff
@@ -834,11 +844,11 @@ def build_lfq_tables(
                     file_format=self.config["search_output"]["file_format"],
                 )
 
-        protein_df_melted = lfq_df.melt(
-            id_vars="pg", var_name="run", value_name="intensity"
-        )
-
-        psm_df = psm_df.merge(protein_df_melted, on=["pg", "run"], how="left")
+        if len(lfq_df) > 0 and len(lfq_df.columns) > 2:
```

Comment:
we are reusing `lfq_df` from above, right? could this be made more robust, or explicit?
(cf. also the comment "# IMPORTANT: 'pg' has to be the last group in the list as this will be reused")

Reply:
Excellent task for claude